<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Docking and Free Positioning Example</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_2">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport_2"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuViews">
    <property name="title">
     <string>Views</string>
    </property>
    <addaction name="actionMode_1"/>
    <addaction name="actionMode_2"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools_2">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp_2">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="menuFile_2"/>
   <addaction name="menuViews"/>
   <addaction name="menuTools_2"/>
   <addaction name="menuHelp_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Mode Selector</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFree_Mode"/>
   <addaction name="actionDock_Mode"/>
  </widget>
  <action name="actionExport">
   <property name="text">
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionReset_Layout">
   <property name="text">
    <string>Reset Layout</string>
   </property>
  </action>
  <action name="actionFree_Mode">
   <property name="text">
    <string>모드1 (자유 배치)</string>
   </property>
  </action>
  <action name="actionDock_Mode">
   <property name="text">
    <string>모드2 (기본 도킹)</string>
   </property>
  </action>
  <action name="actionTest_Option">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test Option</string>
   </property>
  </action>
  <action name="actionTest_Cases">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test Cases</string>
   </property>
  </action>
  <action name="actionTools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionGraph">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plotly Graph</string>
   </property>
  </action>
  <action name="actionExport_2">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionMode_1">
   <property name="text">
    <string>Mode 1</string>
   </property>
  </action>
  <action name="actionMode_2">
   <property name="text">
    <string>Mode 2</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>